examples/0-beginner/02-fine-tuning.py

```diff
@@ -1,198 +1,273 @@
-you want to fine-tune the model on a completely new target, you might need to consider
-the transfer learning approach and introduce a new target in the ``options.yaml`` file.
-More details about this approach can be found in the :ref:`Transfer Learning
-<transfer-learning>` section of the documentation.
+You can check finetuning training curves by parsing the ``train.csv`` that is written
+by ``mtt train``. We remove the old outputs folder from other examples, which
+is not necessary for the normal usage.
 """
+
+# %%
+#
+import glob
+import subprocess
+
+import ase.io
+import matplotlib.pyplot as plt
+import numpy as np
+from metatomic.torch.ase_calculator import MetatomicCalculator
+
+
+# %%
+#
+
+# In order to obtain a pretrained model, you can use a PET-MAD checkpoint from
+# huggingface. Here, we get the PET-MAD ckpt, run ``mtt train`` as a subprocess, and
+# delete the old outputs folder and old model checkpoints.
+
+# %%
+#
+subprocess.run(["rm", "-rf", "outputs", "pet-mad-v1.1.0.ckpt*"], check=True)
+subprocess.run(
+    [
+        "wget",
+        "https://huggingface.co/lab-cosmo/pet-mad/resolve/v1.1.0/models/pet-mad-v1.1.0.ckpt",
+    ],
+    check=True,
+)
+
+subprocess.run(["mtt", "train", "options-ft.yaml", "-o", "model-ft.pt"], check=True)
+
+# %%
+#
+# After training, we can check if finetuning was successful.
+# First we check the training curves, that are saved in ``train.csv`` in the outputs
+# folder. We start with parsing the csv file.
+csv_path = glob.glob("outputs/*/*/train.csv")[-1]
+with open(csv_path, "r") as f:
+    header = f.readline().strip().split(",")
+    f.readline()  # skip units row
+
+# Build dtype
+dtype = [(h, float) for h in header]
+
+# Load data as plain float array
+data = np.loadtxt(csv_path, delimiter=",", skiprows=2)
+
+# Convert to structured
+structured = np.zeros(data.shape[0], dtype=dtype)
+for i, h in enumerate(header):
+    structured[h] = data[:, i]
+
+# %%
+#
+# Now, let's plot the learning curves.
+
+training_energy_RMSE = structured["training energy/finetune RMSE (per atom)"]
+training_forces_MAE = structured["training forces[energy/finetune] MAE"]
+validation_energy_RMSE = structured["validation energy/finetune RMSE (per atom)"]
+validation_forces_MAE = structured["validation forces[energy/finetune] MAE"]
+
+fig, axs = plt.subplots(1, 2, figsize=((8, 3.5)))
+
+axs[0].plot(training_energy_RMSE, label="training energy RMSE")
+axs[0].plot(validation_energy_RMSE, label="validation energy RMSE")
+axs[0].set_xlabel("Epochs")
+axs[0].set_ylabel("energy / meV")
+axs[0].set_xscale("log")
+axs[0].set_yscale("log")
+axs[0].legend()
+axs[1].plot(training_forces_MAE, label="training forces MAE")
+axs[1].plot(validation_forces_MAE, label="validation forces MAE")
+axs[1].set_ylabel("force / meV/A")
+axs[1].set_xlabel("Epochs")
+axs[1].set_xscale("log")
+axs[1].set_yscale("log")
+axs[1].legend()
+plt.tight_layout()
+plt.show()
```

Comment:
Maybe we suppress after this line. It is not helping a lot I think

<img width="837" height="706" alt="Image" src="https://github.com/user-attachments/assets/85b142a4-3389-481e-93e1-0fad9867d3d3" />

Not sure if a `;` is enough....

```suggestion
plt.show();
```

Reply:
No...it's runtime warnings, I think we need to suppress in the `conf.py`